src/github.ts

```diff
@@ -789,17 +809,18 @@ export class GitHub {
 
   async getFileContentsWithDataAPI(
     path: string,
-    defaultBranch: string | undefined
+    branch: string
   ): Promise<GitHubFileContents> {
+    const options: any = {
```

Comment:
if we could figure out a type here might be nice to get rid of the `any`, I wonder if we can pluck something out of `@octokit`.

Reply:
Heh, yeah, I noticed that and was like... how did this pass the linter before? But I think it was introduced originally in the large file PR. I can take a look if you like.

Reply:
It's good I took a look at that, there was a hidden chance for a deref'd `undefined`.